### PR TITLE
feat: introduced 2 functions to handle the business logic

### PR DESCRIPTION
**Pull request by Kavindu-248**

### CODE REVIEW

File: apps/assesment/services.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+from typing import Self
+from rest_framework.exceptions import ValidationError
+from apps.dispensary.models import Order,Prescription, PrescriptionMedicine
+
+
+def create_order(appointment, pharmacy):
+     order = Order.objects.create(appointment=appointment)
+     order.pharmacy_id = pharmacy
```
Comment (Raam124) [inline apps/assesment/services.py:8]: you can pass the pharmacy id when creating why doing it separately

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/services.py (hunk 1 above)
Comment (Kavindu-248) [reply apps/assesment/services.py]: Change made

File: apps/assesment/services.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+from typing import Self
+from rest_framework.exceptions import ValidationError
+from apps.dispensary.models import Order,Prescription, PrescriptionMedicine, Pharmacy
+
+
+def create_order(appointment, pharmacy):
+     order = Order.objects.create(appointment=appointment ,pharmacy_id=pharmacy)
```
Comment (Raam124) [inline apps/assesment/services.py:7]: rename is parameter as pharmacy_idotherwise might get confused with object and id

**Review by Raam124 — CHANGES_REQUESTED** [review]

File: apps/assesment/services.py (hunk 2 above)
Comment (Kavindu-248) [reply apps/assesment/services.py]: Change made

**Review by Raam124 — APPROVED** [review]

**Pull request merged**